src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -44,9 +45,24 @@ public HearingManagementServiceImpl(final MessageSenderConfiguration messageSend
 
     @Override
     public void processRequest(String caseId, HearingDetailsRequest hearingDetailsRequest) {
-        Integer latestVersion = cftHearingService.getLatestVersion(caseId);
+
+        val hearingId = cftHearingService.getLatestVersion(caseId);
```

Comment:
```suggestion
        val latestVersion = cftHearingService.getLatestVersion(caseId);
```
Sorry, this one is still `latestVersion`, but `caseId` is also known as the `hearingId` 😄